cpp/cluster/ClusterProperties.cc

```diff
@@ -76,16 +75,16 @@ void ClusterProperties::compute(const box::Box& box, const vec3<float>* points,
     {
         float s = float(m_cluster_sizes[c]);
         vec3<float> v = m_cluster_centers[c] / s + ref_pos[c];
-        m_cluster_centers[c] = box.wrap(v);
+        m_cluster_centers[c] = nq->getBox().wrap(v);
     }
 
     // Now that we have determined the centers of mass for each cluster, tally
     // up the gyration tensor. This has to be done in a loop over the points.
-    for (unsigned int i = 0; i < Np; i++)
+    for (unsigned int i = 0; i < nq->getNPoints(); i++)
     {
         unsigned int c = cluster_idx[i];
-        vec3<float> pos = points[i];
-        vec3<float> delta = box.wrap(pos - m_cluster_centers[c]);
+        vec3<float> pos = (*nq)[i];
+        vec3<float> delta = nq->getBox().wrap(pos - m_cluster_centers[c]);
```

Comment:
This is the bondVector calculation, but may be a little odd since you'll have to pass in the `cluster_centers` where you'd normally put query_points. May also want to think about proper directionality here; shouldn't it point from the center out to the point? That would affect the bondVector call